lib/actions/query.js

```diff
@@ -1,67 +1,62 @@
 const jsforce = require('jsforce');
-const messages = require('elasticio-node').messages;
-
-/**
- * See list on https://na45.salesforce.com/services/data/
- * @type {string}
- */
-const SALESFORCE_API_VERSION = process.env.SALESFORCE_API_VERSION || '45.0';
+const { messages } = require('elasticio-node');
+const common = require('../common.js');
 
 exports.process = function processAction(msg, conf) {
```

Comment:
Why it is not an `async` function?

Reply:
It looks like an outdated approach. We didn't touch much logic here. We always use `async` functions if our function has asynchronous code as your guideline suggests.